aiogcd/orm/filter.py

```diff
@@ -50,6 +50,27 @@ def __init__(self, model, *filters, has_ancestor=None, key=None):
 
         super().__init__(**filter_dict)
 
+    async def set_offset_limit(self, offset, limit):
```

Comment:
No need for this to be async actually ☺️ this is CPU-bound and doesn't actually await or depend on any IO

Reply:
You are right. Changed to be non-async